xarray/core/dataarray.py

```diff
@@ -3781,6 +3781,8 @@ def polyfit(
         See Also
         --------
         numpy.polyfit
+        numpy.polyval
+        xarray.DataArray.polyval
```

Comment:
doesnt link to polyval page yet

Reply:
that's because the name of the function is `xarray.polyval` not `xarray.DataArray.polyval`:
```suggestion
        xarray.polyval
```